app/main.py

```diff
@@ -1,2 +1,60 @@
 class Dictionary:
-    pass
+    def __init__(self):
+        self.length = 0
+        self.hash_table = [[] for _ in range(8)]
+        self.capacity = len(self.hash_table)
+
+    def __setitem__(self, key, value):
+        self._resize()
+        index = hash(key) % self.capacity
+        iteration = 0
+        while iteration != len(self) + 1:
+            iteration += 1
+            if self.hash_table[index]:
+
+                if hash(key) == self.hash_table[index][0] and\
+                        key == self.hash_table[index][1]:
+                    self.hash_table[index] = [hash(key), key, value]
+                    break
+
+                index += 1
```

Comment:
It can be much more concise. It is relevant to use such incrementation:` index = (index + 1) % self.capacity`.